.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint and Test
+
+on:
+  pull_request:
+  push:
```

Comment:
I think if we won't complete values here, we can remove them and it should take the default values 🤔  is it showing for you as required?

Reply:
That config triggers the workflow for any pull request and push. Required status checks are managed through the branch protection rules.